src/bundles/page-header.js

```diff
@@ -1,111 +1,62 @@
 
-function PageHeader({ onUiLanguageClick }) {
-  const user = useUser();
+const generateCookieHash = textInAllLanguages => {
+  return textInAllLanguages ? md5(JSON.stringify(textInAllLanguages)) : '';
+};
+
+function PageHeader() {
   const settings = useSettings();
-  const { uiLanguage } = useLocale();
-  const { t } = useTranslation('page');
-  const helpPage = settings?.helpPage?.[uiLanguage];
+  const { announcementCookieNamePrefix } = useService(ClientConfig);
+
+  const [showAnnouncement, setShowAnnouncement] = useState(false);
 
-  const pageMenuItems = [
-    {
-      key: 'dashboard',
-      label: t('pageNames:dashboard'),
-      icon: <DashboardIcon />,
-      onClick: () => { window.location = routes.getDashboardUrl(); },
-      showWhen: !!user
-    },
-    {
-      key: 'redaction',
-      label: t('pageNames:redaction'),
-      icon: <EditIcon />,
-      onClick: () => { window.location = routes.getRedactionUrl(); },
-      showWhen: hasUserPermission(user, permissions.MANAGE_CONTENT)
-    },
-    {
-      key: 'admin',
-      label: t('pageNames:admin'),
-      icon: <SettingsIcon />,
-      onClick: () => { window.location = routes.getAdminUrl(); },
-      showWhen: hasUserPermission(user, permissions.ADMIN)
-    },
-    {
-      key: 'help',
-      label: helpPage?.linkTitle,
-      icon: <QuestionOutlined />,
-      onClick: () => { window.location = helpPage ? routes.getDocUrl({ id: helpPage.documentId }) : ''; },
-      showWhen: !!helpPage?.documentId
-    },
-    {
-      key: 'ui-language',
-      label: t('common:language'),
-      icon: <LanguageIcon />,
-      onClick: () => onUiLanguageClick(),
-      showWhen: true
-    },
-    {
-      key: 'logout',
-      label: t('common:logOut'),
-      icon: <LogoutIcon />,
-      onClick: () => { window.location = routes.getLogoutUrl(); },
-      showWhen: !!user
+  const announcementCookieName = `${announcementCookieNamePrefix}_${useMemo(() => generateCookieHash(JSON.stringify(settings.announcement)), [settings.announcement])}`;
+
+  useEffect(() => {
+    const announcementCookie = getCookie(announcementCookieName);
+    if (!announcementCookie && settings.announcement?.text) {
+      setShowAnnouncement(true);
     }
-  ].filter(item => item.showWhen);
+  }, [announcementCookieName, settings.announcement]);
 
-  const handleMenuItemClick = ({ key }) => {
-    const clickedItem = pageMenuItems.find(item => item.key === key);
-    clickedItem.onClick();
+  const handleAnnouncementClose = () => {
+    setLongLastingCookie(announcementCookieName, 'true');
+    setShowAnnouncement(false);
   };
 
-  const menuItems = pageMenuItems.map(({ key, label, icon }) => ({ key, label, icon }));
-
   return (
     <header className="PageHeader">
-      <div className="PageHeader-header">
-        <div className="PageHeader-headerContent PageHeader-headerContent--left">
-          <HeaderLogo />
+      <div className="PageHeader-content">
+        <HeaderLogo />
```

Comment:
If we have the time (and that is a bug IF) we could make the header logo injectable, like we do the  site logo, so that OMA does not need to set up the custom `page-header` component and as consequence then it also does not need to set up the `page-template` component. It can use the defaults that `educandu` provides for both.

Reply:
I'm all for it. Actually let's do it now?

Reply:
I'd maybe push it to the feature freeze week 🤔 It's something nice to do then. Will write it down in the feature freeze epic.

Reply:
Agreed!